kong/runloop/plugin_servers/init.lua

```diff
@@ -124,7 +132,7 @@ local exposed_api = {
   end,
 
   ["kong.nginx.req_start_time"] = function()
-    local saved = save_for_later[coroutine_running()]
+    local saved = get_saved()
```

Comment:
I created another PR #9551 and changed it,
so I think in this PR you need not care about it.